src/modules/navigator/navigator_main.cpp

```diff
@@ -210,6 +210,8 @@ Navigator::run()
 
 			// update parameters from storage
 			params_update();
+
+			_gf_breach_avoidance.updateParameters();
```

Comment:
Instead, pass `this` to `GeofenceBreachAvoidance`: `GeofenceBreachAvoidance(ModuleParams* parent) : ModuleParams(parent) {}`
```suggestion
```